spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/PubSubAdmin.java

```diff
@@ -196,8 +196,9 @@ public Subscription createSubscription(String subscriptionName, String topicName
 
 		int finalAckDeadline = this.defaultAckDeadline;
 		if (ackDeadline != null) {
-			Assert.isTrue(ackDeadline >= 0,
-					"The acknowledgement deadline value can't be negative.");
+			Assert.isTrue(ackDeadline >= 10 && ackDeadline <= 600,
```

Comment:
Should these two special numbers be made into constants ?

Reply:
I'll refactor them into constants

Reply:
Something like [this](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/27c90f1298bb27a618ff59b3cd17321c577f3a6b/google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java#L97).
Unfortunately it's package private there.